release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2994,6 +2994,10 @@ $ oc adm release info 4.11.16 --pullspecs
 
 * With this release, when you xref:../installing/installing_gcp/uninstalling-cluster-gcp.adoc#cco-ccoctl-deleting-sts-resources_uninstalling-cluster-gcp[delete GCP resources with the Cloud Credential Operator utility], you must specify the directory containing the files for the component `CredentialsRequest` objects.
 
+[id="ocp-4-11-16-bug-fixes"]
+==== Bug fixes
+* Previously, if you used upper-case letters when specifying an Azure Disk Encryption Set (DES) or Resource Group (RG) name, the validation would fail. With this release, you can now use upper and lower case letters in DES and RG names. (link:https://issues.redhat.com//browse/OCPBUGS-4826[*OCPBUGS#4626*])
```

Comment:
```suggestion
* Previously, if you used upper-case letters when specifying an Azure Disk Encryption Set (DES) or Resource Group (RG) name, the validation would fail. With this release, you can now use upper and lower case letters in DES and RG names. (link:https://issues.redhat.com//browse/OCPBUGS-4826[*OCPBUGS#4826*])
```